xrdsst/controllers/endpoint.py

```diff
@@ -4,19 +4,18 @@
 from xrdsst.controllers.base import BaseController
 from xrdsst.controllers.service import ServiceController
 from xrdsst.controllers.client import ClientController
-from xrdsst.models import Endpoint, ServiceType
+from xrdsst.models import Endpoint, ServiceType, ServiceClients
 from xrdsst.rest.rest import ApiException
 from xrdsst.resources.texts import texts
 
-
 class EndpointController(BaseController):
     class Meta:
         label = 'endpoint'
         stacked_on = 'base'
         stacked_type = 'nested'
         description = texts['endpoint.controller.description']
 
-    @ex(label='add-endpoints', help="Add endpoints", arguments=[])
+    @ex(help="Add endpoints", arguments=[])
```

Comment:
If it is not too difficult, please change this to "add" like in ClientController, because "xrdsst endpoints add" would probably look better than "xrdsst endpoints add-endpoint":

    @ex(help="Add client subsystem", arguments=[])
    def add(self):